18/election_transaction.yaml

```diff
@@ -0,0 +1,35 @@
+---
+"$schema": "http://json-schema.org/draft-04/schema#"
+type: object
+title: Election Schema
+required:
+- operation
```

Comment:
`operation` is not defined in the spec

Reply:
`operation` is defined for particular election types.

Reply:
Ah, yes, makes sense! I would also add a description.